README.md

```diff
@@ -3,16 +3,28 @@
 If your team works on a Slack workspace and the team has to maintain a bunch of different email ids and wish to
 be notified of every email instantly on slack; there is a hack for you.
 
-# How does it work ?
+# How does it work?
+
+This bot watches for messages in the slackbot chat and whenever a message arrives with an attached email, It forwards it to a channel as a message.
+
+# How to make it work ?
 1. Set up a forwarding email. If you send any email to this address, it will appear as a file in your Direct Messaging channel with slackbot.
-1. Add this address as a forwarding address of your email account. (For gmail find forwarding options from gmail settings)
-1. Use Slack API to get notified about this.
-1. Use Slack API to post the email on a channel you want to.
+2. Add this address as a forwarding address of your email account. (For gmail find forwarding options from gmail settings)
```

Comment:
It is actually a good idea to use `1.` for all the list items. Adding or removing items gets easier without making us change all the indices.

Reply:
Hmm I never knew this syntax. Updating code :+1: